draft-irtf-cfrg-vdaf.md

```diff
@@ -379,54 +379,264 @@ exception that Clients submitting measurements may be anonymous.
+desired aggregate result:
+
+* `Daf.out_shares_to_agg_share(agg_param: AggParam, out_shares: Vec[OutShare])
+  -> agg_share: AggShare` is the deterministic aggregation algorithm. It is run
+  by each Aggregator over the output shares it has computed over a batch of
+  measurement inputs.
+
+~~~~
+    Aggregator 0    Aggregator 1        Aggregator SHARES-1
+    ============    ============        ===================
+
+    out_share_0_0   out_share_1_0       out_share_[SHARES-1]_0
+    out_share_0_1   out_share_1_1       out_share_[SHARES-1]_1
+    out_share_0_2   out_share_1_2       out_share_[SHARES-1]_2
+         ...             ...                     ...
+    out_share_0_B   out_share_1_B       out_share_[SHARES-1]_B
+      |               |                   |
+      V               V                   V
+    +-----------+   +-----------+       +-----------+
+    | out2agg   |   | out2agg   |   ... | out2agg   |
+    +-----------+   +-----------+       +-----------+
+      |               |                   |
+      V               V                   V
+    agg_share_0     agg_share_1         agg_share_[SHARES-1]
+~~~~
+{: #aggregate-flow title="Aggregation of output shares. `B` indicates the number of
+measurements in the batch."}
+
+For simplicity, we have written this algorithm in a "one-shot" form, where all
+output shares for a batch are provided at the same time. Many DAFs may also
+support a "streaming" form, where shares are processed one at a time.
+
+> OPEN ISSUE: It may be worthwhile to explicitly define the "streaming" API. See
+> issue#47.
+
+## Unsharding {#sec-daf-unshard}
+
+After the Aggregators have aggregated a sufficient number of output shares, each
+sends its aggregate share to the Collector, who runs the following algorithm to
+recover the following output:
+
+* `Daf.agg_shares_to_result(agg_param: AggParam, agg_shares: Vec[AggShare]) ->
+  AggResult` is run by the Collector in order to compute the aggregate result
+  from the Aggregators' shares. The length of `agg_shares` MUST be `SHARES`.
+  This algorithm is deterministic.
+
+~~~~
+    Aggregator 0    Aggregator 1        Aggregator SHARES-1
+    ============    ============        ===================
+
+    agg_share_0     agg_share_1         agg_share_[SHARES-1]
+      |               |                   |
+      V               V                   V
+    +-----------------------------------------------+
+    | agg_shares_to_result                          |
+    +-----------------------------------------------+
+      |
+      V
+    agg_result
+
+    Collector
+    =========
+~~~~
+{: #unshard-flow title="Computation of the final aggregate result from aggregate
+shares."}
+
+> QUESTION Maybe the aggregation algorithms should be randomized in order to
+> allow the Aggregators (or the Collector) to add noise for differential
+> privacy. (See the security considerations of {{?I-D.draft-gpew-priv-ppm}}.) Or
```

Comment:
update this reference to `draft-ietf-ppm-dap` (and presumably elsewhere in the document where `draft-gpew-priv-ppm` occurs)